persistence/src/main/java/cz/fi/muni/pa165/model/dao/UserDaoImpl.java

```diff
@@ -21,6 +23,8 @@
     @PersistenceContext
     private EntityManager em;
 
+    final static Logger log = LoggerFactory.getLogger(UserDaoImpl.class);
```

Comment:
Autowire LoggerFactory (constructor) and then acquire proper logger there. This is called service locator pattern; you cannot change logging functionality in tests for example. This makes things harder.